### PR TITLE
feat: standardize view button text

### LINKED ISSUE #46497: Making `/settings`' buttons consistent
### Is your feature request related to a problem? Please describe.

There are 3 buttons with different innerText `View`, `Show Code`, `Show Solution`

~~Edit: did I just put same buttons twice🤦🤣, because I am currently away from laptop~~ check [setting](https://www.freecodecamp.org/settings)

image
image

### Describe the solution you'd like

Swapping `Show Code` and `Show Solution` with `View` for consistent look,  while adding `.sr-only` to `span` in the `button` with `innerText` equal to **Code** or **Solution**.

Examples:

``` diff
- <button>Show Code</button>
- <button>Show Solution</button>

+ <button>View <span class="sr-only>Code</span></button>
+ <button>View <span class="sr-only>Solution</span></button>
```

### Describe alternatives you've considered

Nothing, this isn't a true issue. Quality of life change

### Additional context

This idea is originated from full button width

for more info check comment in https://github.com/freeCodeCamp/freeCodeCamp/issues/45947#issuecomment-1130030952

### PR DESCRIPTION
**Pull request by Hdanzo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46497

Add new translation keys 
Delete unused translation keys 
Fix tests broken with new functionalities

### CODE REVIEW

**Comment by Sboonny** [comment]
I have closed this and reopened it, because cypress test looks weird.

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Hdanzo, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
     "edit": "Edit",
-    "show-code": "Show Code",
-    "show-solution": "Show Solution",
-    "show-project": "Show Project",
     "frontend": "Front End",
     "backend": "Back End",
     "view": "View",
+    "view-code": "View <0>Code</0>",
+    "view-solution": "View <0>Solution</0>",
+    "view-project": "View <0>Project</0>",
```
Comment (Sboonny) [inline client/i18n/locales/english/translations.json:14]: There is may cause confusion to screen readers, and translation team

```suggestion
```

File: client/src/components/solution-display-widget/index.tsx [hunk 2]
```diff
@@ -27,9 +27,13 @@ export function SolutionDisplayWidget({
...
-    displayContext === 'settings'
-      ? t('buttons.show-solution')
-      : t('buttons.view');
+    displayContext === 'settings' ? (
+      <Trans i18nKey='buttons.view-solution'>
+        <span className='sr-only'>Solution</span>
+      </Trans>
+    ) : (
+      t('buttons.view')
+    );
```
Comment (Sboonny) [inline client/src/components/solution-display-widget/index.tsx:36]: with deletion of other view variation, we can have simple translated view text
```suggestion
 const viewText = t('buttons.view')
```

File: client/src/components/solution-display-widget/index.tsx [hunk 3]
```diff
@@ -131,6 +142,7 @@ export function SolutionDisplayWidget({
       href={solution}
       id={`btn-for-${id}`}
       rel='noopener noreferrer'
+      title='project link'
       target='_blank'
     >
       {showOrViewText}
```
Comment (Sboonny) [inline client/src/components/solution-display-widget/index.tsx:148]:
```suggestion
      {viewText}

File: client/src/components/solution-display-widget/index.tsx [hunk 4]
```diff
@@ -70,10 +74,13 @@ export function SolutionDisplayWidget({
...
       className='btn-invert'
       data-cy={dataCy}
+      title='view code button'
       id={`btn-for-${id}`}
       onClick={showUserCode}
     >
-      {t('buttons.show-code')}
+      <Trans i18nKey='buttons.view-code'>
+        <span className='sr-only'>Code</span>
+      </Trans>
```
Comment (Sboonny) [inline client/src/components/solution-display-widget/index.tsx:83]:
```suggestion
      {viewText}
```

File: client/src/components/solution-display-widget/index.tsx [hunk 5]
```diff
@@ -86,10 +93,14 @@ export function SolutionDisplayWidget({
         title={t('buttons.view')}
       >
         <MenuItem bsStyle='primary' onClick={showUserCode}>
-          {t('buttons.show-code')}
+          <Trans i18nKey='buttons.view-code'>
+            <span className='sr-only'>Code</span>
+          </Trans>
```
Comment (Sboonny) [inline client/src/components/solution-display-widget/index.tsx:98]:
```suggestion
      {viewText}
```

File: client/src/components/solution-display-widget/index.tsx [hunk 6]
```diff
@@ -86,10 +93,14 @@ export function SolutionDisplayWidget({
...
         </MenuItem>
         <MenuItem bsStyle='primary' onClick={showProjectPreview}>
-          {t('buttons.show-project')}
+          <Trans i18nKey='buttons.view-project'>
+            <span className='sr-only'>Project</span>
+          </Trans>
```
Comment (Sboonny) [inline client/src/components/solution-display-widget/index.tsx:103]:
```suggestion
      {viewText}
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
I had a meeting with @ahmadabdolsaheb, and we discussed the pull request and there are issues and I didn't notice

File: client/src/components/solution-display-widget/index.tsx [hunk 7]
```diff
@@ -70,10 +74,13 @@ export function SolutionDisplayWidget({
       bsStyle='primary'
       className='btn-invert'
       data-cy={dataCy}
+      title='view code button'
```
Comment (Sboonny) [inline client/src/components/solution-display-widget/index.tsx:77]: Adding a title will add a tooltip, if we are going to keep it around, we need it to be descriptive of what the button for, like `navigate to solution of this certifcation` and it needs to be translatable.

we can cross that bridge another time, so here is my suggestion

```suggestion
```

If you want to add it, you can add descriptive title in `.json` file and use it as a value for the title. `viewText` is a good example

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**